CMake/mlpack.cmake

```
@@ -592,6 +592,54 @@ macro(find_mlpack_internal)
 
 endmacro()
 
+macro(compile_OpenBLAS)
+  if (CMAKE_SYSTEM_NAME STREQUAL "Windows")
+    set(OPENBLAS_SRC_DIR ${CMAKE_BINARY_DIR}/deps/OpenBLAS-${OPENBLAS_VERSION})
+    set(OPENBLAS_BUILD_DIR ${OPENBLAS_SRC_DIR}/build)
+    set(OPENBLAS_OUTPUT_LIB_DIR ${OPENBLAS_BUILD_DIR}/lib)
+    # always compile BLAS as release.
+    set(BLASS_BUILD_TYPE "Release")
```

Should this be `BLAS_BUILD_TYPE`?

```suggestion
    set(BLAS_BUILD_TYPE "Release")
```